Tribler/Test/Community/Tunnel/test_community.py

```diff
@@ -57,6 +57,7 @@ def create_node(self):
                                       working_directory=u":memory:")
         mock_ipv8.overlay.bandwidth_wallet = TrustchainWallet(overlay)
         mock_ipv8.overlay.dht_provider = MockDHTProvider(mock_ipv8.endpoint.wan_address)
+        self.private_nodes.append(mock_ipv8)
```

Comment:
What's the reason for this change?

Reply:
`tearDown` in this test class unloads the `private_nodes`. `create_node` is used several times through this file, so each instance should be tracked to be properly unloaded. Before the patch it was only registered by hand at line 85. Now it is done automatically.

Reply:
Ok thanks for the explanation!

Reply:
But, create node is also used by `initialize`, so now all the nodes in `self.nodes` are also in `self.private_nodes`.